decidim-proposals/spec/commands/decidim/proposals/admin_update_proposal_category_spec.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+module Decidim
+  module Proposals
+    module Admin
+      describe UpdateProposal do
+        describe "call" do
+          let(:organization) { create(:organization) }
+          let(:participatory_process) { create(:participatory_process, organization: organization) }
+          let(:feature) { create(:proposal_feature, participatory_space: participatory_process) }
+          let!(:category_one) { create :category, participatory_space: participatory_process }
+          let!(:category) { create :category, participatory_space: participatory_process }
```

Comment:
Instead of creating all the ancestors you can skip it and just use `proposal.feature.participatory_space`

Reply:
thanks for the tip